float32 x_error # absolute value of the x error
float32 y_error # absolute value of the x error
float32 theta_error # absolute value of the angular error
